docs/l10n.rst

```diff
@@ -154,6 +164,59 @@ to make adjustments should it suit their particular language.
 .. _terms: https://projectfluent.org/fluent/guide/terms.html
 .. _mozilla-l10n/www-l10n: https://github.com/mozilla-l10n/www-l10n
 
+Variables
+~~~~~~~~~
+
+Single hash comments are applied only to the string immediately following them. They should be used to provide
+additional context for the translators including:
+
+1. What the values of variables are.
+2. Context about where string appears on the page if it is not visible or references other elements on the page.
+3. Explanations of English idioms and jargon that may be confusing to non-native speakers.
+
+.. code-block:: fluent
+
+    # Variables:
+    #   $savings (string) - the percentage saved from the regular price, not including the % Examples: 50, 70
+    example-bundle-savings = Buy now for { $savings }% off.
+
+    # Context: Used as an accessible text alternative for an image
+    example-bookmark-manager-alt = The bookmark manager window in { -brand-name-firefox }.
+    # Context: This lists the various websites and magazines who have mentioned Firefox Relay.
+    # Example: "As seen in: FORBES magainze and LifeHacker"
+    example-social-proof = As seen in:
+
+    example-privacy-on-every = Want privacy on every device?
+    # "You got it" here is a casual answer to the previous question, "Want privacy on every device?"
+    example-you-got-it = You got it. Get { -brand-name-firefox } for mobile.
+
+
+HTML with attributes
+********************
+
+When passing HTML tags with attributes into strings for translation, remove as much room for error as possible by
+putting all the attributes and their values in a single variable. (This is most common with links and their href
+attributes but we do occasionally pass classes with other tags.)
+
+.. code-block:: fluent
+
+    # Variables:
+    #   $attrs (attrs) - link to https://www.mozilla.org/about/
+    example-created = { -brand-name-firefox } was created by <a {$attrs}>{ -brand-name-mozilla }</a>.
+
+    # Variables:
+    #   $class (string) - CSS class used to replace brand name with wordmark logo
+    example-firefox-relay = Add <span { $class }">{ -brand-name-firefox-relay }</span>
```

Comment:
There's an extra double quote in the example, which is exactly one of the problems we had ;-)